subcommand/lifecycle-sidecar/command.go

```diff
@@ -56,16 +64,24 @@ func (c *Command) Run(args []string) int {
 	if err := c.flagSet.Parse(args); err != nil {
 		return 1
 	}
+
 	syncPeriod, logLevel, err := c.validateFlags()
 	if err != nil {
 		c.UI.Error("Error: " + err.Error())
 		return 1
 	}
+
 	logger := hclog.New(&hclog.LoggerOptions{
 		Level:  logLevel,
 		Output: os.Stderr,
 	})
 
+	// Log initial configuration
+	logger.Info("Command configuration", "service-config", c.flagServiceConfig,
+		"consul-location", c.flagConsulLocation,
+		"sync-period", syncPeriod,
+		"log-level", logLevel)
+
 	// Set up Consul client (may already exist in tests).
 	if c.consulClient == nil {
```

Comment:
Do we still need the consul client now that we're shelling out?

Reply:
Removed.